Framework/Kernel/src/PropertyManager.cpp

```diff
@@ -158,7 +158,9 @@ void PropertyManager::splitByTime(std::vector<SplittingInterval> &splitter,
 
     // Now the property does the splitting.
     bool isProtonCharge = prop->name() == "proton_charge";
-    prop->splitByTime(splitter, output_properties, isProtonCharge);
+    if (auto timeSeriesProperty = dynamic_cast<ITimeSeriesProperty *>(prop)) {
+      timeSeriesProperty->splitByTime(splitter, output_properties, isProtonCharge);
```

Comment:
As far as I understand, we don't need `timeSeriesProperty::splitByTime()` as well as owner functions `PropertyManager::splitByTime()` and `LogManager::splitByTime()` because we're not splitting the time series anymore. Am I misreading this?
If I am right and these functions need to go, should we leave it for a cleaning Task?

Reply:
The logs themselves shouldn't get split anymore, but that is being done in #35380.